lib/api/helpers.js

```diff
@@ -4,12 +4,22 @@ import { FlowRouter } from "meteor/kadira:flow-router-ssr";
 import { Shops } from "/lib/collections";
 
 /* eslint no-unused-vars: 0 */
-//
-//  TODO review this slugify import in lib/api/helpers
-//
+
+// Export Reaction using commonJS style for common libraries to use Reaction
+// https://docs.meteor.com/packages/modules.html#CommonJS
+let Reaction;
```

Comment:
Actually, it just occurred to me that pulling the Collections/Schemas into there and removing from the client/server locations would be a breaking change for anyone not importing `Reaction` from `/lib/api`.  I think we need to leave that part alone for now.